docs/navigation-configuration.md

```diff
@@ -268,3 +268,15 @@ Luigi.setConfig({
     ]
   }
 });
+```
+
+## View Groups
+You can use the view groups feature to override the default iframes management policy. Imagine you have in your app the following microfrontend views: `http://mysite.com/a#e` and  `http://mysite.com/b#f`. Due to hash routing they don´t belong to the same domain, since the domain is the path until the `#` and therefore they will be rendered in different iframes by default. Nevertheless both views might have the **same origin** (in the example `mysite.com`) and belong to the **same microfrontend** and you will want to render them in the same iframe. The view groups feature makes that possible just by setting the `viewGroup` parameter in the most upper nodes, as their children nodes are automatically considered part of the same view group. 
```

Comment:
The term 'domain' usually refers to the host name only (mysite.com in this case). Luigi's default policy to determine if to render the previous and next view in the same iframe compares the whole url with protocol, domain name, port and even the path till # character - that is true, but I would not refer to it as 'domain', this could be confusing.

Reply:
thanks, done